ipn/ipnlocal/local.go

```diff
@@ -2261,7 +2261,7 @@ func (b *LocalBackend) WatchNotifications(ctx context.Context, mask ipn.NotifyWa
 			ini.SessionID = sessionID
 			ini.State = ptr.To(b.state)
 			if b.state == ipn.NeedsLogin {
-				ini.BrowseToURL = ptr.To(b.authURLSticky)
+				ini.BrowseToURL = ptr.To(b.authURL)
```

Comment:
should this be `authURLToPop`? or do you want to keep existing behavior in this PR and then fix in a follow up?

Reply:
can also do 2 commits